src/hotspot/share/code/codeCache.cpp

```diff
@@ -1801,7 +1801,7 @@ void CodeCache::write_perf_map(const char* filename) {
 
   fileStream fs(filename, "w");
   if (!fs.is_open()) {
-    log_warning(codecache)("Failed to create %s for perf map", filename);
+    st->print_cr("Failed to create %s for perf map", filename);
```

Comment:
Hi -- as this used to be "log_warning" and print something like:
  [1129077.636s][warning][codecache] Failed to create /x for perf map
..it should probably say:
  Warning: Failed to...etc...